Automattic/vip-cli
# Add custom WP config for dev environment

## Description

When spinning up a new dev environment from the CLI, lando would use the default WP configuration that would come with the image. That was a bit problematic, because if we needed to do some tweaks on that configuration, we could not do that.

This was the case with the `WPCOM_IS_VIP_ENV` variable, which should be false as this is a local environment, but was true by default.

This PR adds the `wp-config-defaults.php` file, which can be customized further to fit a local development environment. In this case, the only change from the default behavior is `WPCOM_IS_VIP_ENV`.

## Steps to Test

Outline the steps to test and verify the PR here.

1. Check out PR
2. Run `npm link` to install the latest version
3. Spin up a dev environment and create/edit a post (doing that before wouldn't work due to the incorrect value of `WPCOM_IS_VIP_ENV`)
